cmd/test/test.go

```diff
@@ -4,12 +4,9 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var (
-	debug  bool
-	format string
-)
+var ()
 
-func CreateTestCommand() *cobra.Command {
+func CreateCommand() *cobra.Command {
```

Comment:
Was `CreateTestCommand` causing an issue?

Reply:
Renamed to align with all of the other wrapper command method names